src/doc/book.toml

```
@@ -4,3 +4,4 @@ author = "Alex Crichton, Steve Klabnik and Carol Nichols, with Contributions fro
 
 [output.html]
 git-repository-url = "https://github.com/rust-lang/cargo/tree/master/src/doc/src"
+edit-url-template = "https://github.com/rust-lang/cargo/tree/master/src/doc/src/{path}"
```

I believe the path is relative to the book root.

```suggestion
edit-url-template = "https://github.com/rust-lang/cargo/tree/master/src/doc/{path}"
```

Also, is there a particular reason you chose to not go to the `edit` link? (Not that I have a strong preference, but that was the intended way to use this.)

---

No, just missed that!